Descriptions/Curation.md

```diff
@@ -0,0 +1,62 @@
+>Depending on the task, it is not always relevant for salts (2 fragments, one of them is the ion of one atom). Salts should be handled by taking the major fragment and following neutralization (removing of charge) for the majority of molecular target associated models (like hERG) and not for properties.
+4. Neutralization: For most tasks, we neutralize molecules as their charge will significantly depend on biological/chemical environment.
+5. Standardization of structural formulas: Different chemotypes can be represented differently in various chemical notations. We perform standardization to obtain equal descriptors/fingerprints derived from such groups.
+6. Equal aromatization: We transform the whole dataset to an equal aromatic or kekule form.
+7. Equal normalization: For all macromolecular target activities and for the majority of properties (except pKa), we perform normalization (not to be confused with neutralization or reionization). Normalization conserves the total charge of the molecule.
+8. Equal tautomerization: In most cases, the equal tautomeric form of groups should be present in the whole dataset to avoid different descriptors/fingerprints generation. It is legitimized as both forms are usually present in real systems but lead to different descriptors/fingerprints generations.
+9. Handling stereo information: We conserve stereo information only when all the data contains it, otherwise we exclude it.
+10. Handling hydrogen information: The presence or absence of hydrogens in the structures leads to unequal descriptors/fingerprints generation. We exclude or add explicit hydrogens to all the structures in the dataset depending on the analyzed problem.
+
+## Endpoint Data Curation
+
+Another major stage of curation deals with experimental numerical data. For this, we focus on the following:
+1. Removal of incorrectly annotated data/irrelevant data: We remove such data points if they are not specific to the model. For example, if developing a model for SARS-CoV-2 inhibitors, we exclude all entries for SARS-CoV, MERS, or SARS-CoV-2 mutants.
+2. Duplicates handling: Mixed datasets often contain several property/activity values for a single structure. We explore the distribution of these entries and exclude outliers. We usually take the median, and for concentration values, the harmonic mean or median of logarithmic transformed values are also accepted.
+>For modeling tasks, duplicates are not just equal structures. We compare and search for duplicates of descriptors/fingerprints vectors.
+3. Activity cliffs handling: The majority of machine learning approaches use smooth functions as a model. The last are fitted to capture the data, although structures highly similar to others but with a high difference in property/activity can give a strong bias to parameter estimates. We exclude them when there is just a single activity cliff in the subgroup.
```

Comment:
We decided that it is hard to do automatically? Should we really include this step?